tidb.go

```diff
@@ -243,9 +243,24 @@ func NewStore(uri string) (kv.Storage, error) {
 
 var queryStmtTable = []string{"explain", "select", "show", "execute", "describe", "desc"}
 
+func trimSQL(sql string) string {
```

Comment:
Yes, it is my fault.
SQL like `select * /* comment */ from t;` is supported now.
Seems we only not support start with '/*' sql, like `/* comment */ select * from t`